scripts/generator/graphql_generator/csharp/SDK/iOS/ApplePayEventResponse.cs.erb

```diff
@@ -15,6 +15,14 @@ namespace <%= namespace %>.SDK.iOS {
             SummaryItems = summaryItems ?? new List<SummaryItem>();
             ShippingMethods = shippingMethods ?? new List<ShippingMethod>();
         }
+
+        public string ToJsonString() {
+            var dict = new Dictionary<string, object>();
+            dict["SummaryItems"] = SummaryItems;
```

Comment:
It's unfortunate but it's not as easy as this. `SummaryItems` would need to be turned into a `List<Dictionary<string, object>`.